angularSTEP/src/app/converter.ts

```diff
@@ -0,0 +1,38 @@
+import {QueryDocumentSnapshot, SnapshotOptions} from '@angular/fire/firestore';
+
+import {User} from './user';
+
+export class Converter {
+  public converter = {
+    toFirestore: function (user: User) {
```

Comment:
I assume these functions allow for storage and retrieval of info. Not to familiar, this isn't a problem just could use some info on the process.

Reply:
Yes they are for storage and retrieval of info. I will put comments for the code! I also mentioned it in the first comment on this PR.
The User class and converter class are what you can use to get profile info from Firestore. It takes a document and then puts it into a User instance. The converter class just holds the needed converter.